ui/src/groups/AddGroup/InvitedGroupsDisplay.tsx

```diff
@@ -0,0 +1,20 @@
+import GroupReference from '@/components/References/GroupReference';
+import { usePendingGangsWithoutClaim } from '@/state/groups';
+
+export default function InvitedGroupsDisplay({
+  selectFlag,
+}: {
+  selectFlag: (flag: string) => void;
+}) {
+  const pendingGangs = usePendingGangsWithoutClaim();
+  return (
+    <div className="h-full">
+      <h3 className="mb-4 text-lg font-bold">Pending Invites</h3>
+      <div className="h-5/6 overflow-auto">
+        {Object.keys(pendingGangs).map((flag, index) => (
+          <GroupReference key={index} flag={flag} customOnClick={selectFlag} />
```

Comment:
Something weird going on here:

<img width="1317" alt="Screenshot 2024-01-23 at 5 30 08 PM" src="https://github.com/tloncorp/landscape-apps/assets/5466421/a9499dee-8221-4593-bca4-43e8d78eff24">

clicking blue button does this:

<img width="1096" alt="Screenshot 2024-01-23 at 5 30 50 PM" src="https://github.com/tloncorp/landscape-apps/assets/5466421/7317b164-d69d-42ea-8303-be951313dea2">